src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -42,17 +42,17 @@ Where you create a `FlutterEngine` depends on your host app.
 {% tabs "darwin-framework" %}
 {% tab "SwiftUI" %}
 
-In this example, we create a `FlutterEngine` object inside a SwiftUI `ObservableObject`. 
-We then pass this `FlutterEngine` into a `ContentView` using the 
- `environmentObject()` property. 
+In this example, we create a `FlutterEngine` object inside a SwiftUI [Observable](https://developer.apple.com/documentation/observation/observable) object called `FlutterDependencies`. 
```

Comment:
Hey as part of this change could you tweak [line 11](https://github.com/flutter/website/pull/10888/files#diff-b4b869d276ce91432a0be926d482945629735ae6a5fdb68eb6c9a1f5190d9054R11)?

```diff
- To launch a Flutter screen from an existing iOS, you start a
+ To launch a Flutter screen from an existing iOS app, you start a

Reply:
done